drivers/SmartThings/zigbee-lock/src/samsungsds/init.lua

```diff
@@ -49,10 +50,18 @@ local function unlock_cmd_handler(driver, device, command)
 end
 
 local device_added = function(self, device)
+  lock_utils.populate_state_from_data(device)
   device:emit_event(capabilities.lock.lock.unlocked())
   device:emit_event(capabilities.battery.battery(100))
 end
 
+local battery_init = battery_defaults.build_linear_voltage_init(4.0, 6.0)
```

Comment:
you could just inline this in `init`

unless it's used elsewhere in the driver I can't see in the web diff?

Reply:
That would end up with a duplicate function per device where this is just a single function for the driver.  So just a minor optimization 🤷